packages/scene-composer/src/interfaces/interfaces.tsx

```diff
@@ -148,3 +149,7 @@ export interface IFogSettings {
   near: number;
   far: number;
 }
+
+export interface ISceneBackgroundSetting {
+  colorHex?: number;
```

Comment:
do we need to store the hex number or we can just store the hex string? looks like the color picker returns the string, and the threejs color can handle color string. and we will need to do some extra conversion between color number and string. can we just save the color hex string then?

I found many instances in our code that we allow color to be either number or string, then we add many conversion steps, but the actual value we get from the picker and set to the 3D component can just be a color string generically. then we can simplify some conversion code.

Same comment for the `colorHex` in `IFogSettings`

Reply:
Let me give this ago, as I agree that the string to number conversion is annoying. I think when I looked at the three.js docs only saw hex values as numbers and strings as 'blue' etc. based on the examples but the full spec makes it look like it also takes strings of format '#ff00ff'

Reply:
Swapped fog & scene background to use color: string